File: Python Code/Daily_Flash_Code_0/Week 5/13 Feb/27-DailyFlash_Solutions/27-DailyFlash_Solutions/13_Feb_Solution_Three/C++/prog1.cpp
#include<iostream>

int main() {
	int num;
	std::cout << "Enter num : ";
	std::cin >> num;
	int temp = num, rem, sum = 0, fact;
	while(temp > 0) {
		rem = temp % 10;
		fact = 1;
		for(int i = 1; i <= rem; i++) {
			fact = fact * i;
		}
		sum += fact;
		temp /= 10;
	}
	if(num == sum) {
		std::cout << "Strong num";
	} else {
		std::cout << "Not strong num";
	}

	return 0;
}

File: Python Code/Daily_Flash_Code_0/Week 5/13 Feb/27-DailyFlash_Solutions/27-DailyFlash_Solutions/13_Feb_Solution_Three/Python/prog1.py
num = int(input("Num : "))
temp = num
s = 0
while(temp > 0):
    rem = temp % 10;
    fact = 1
    for i in range(1, rem + 1):
        fact *= i
    s += fact
    temp = int(temp / 10)
if(s == num):
    print("Strong")
else:
    print("not strong")
